upgrade-2.x.sh

```diff
@@ -302,6 +302,50 @@ function pre_update_fix_bootfiles_hook {
     mount --bind "$bootfiles_temp"  /etc/hostapp-update-hooks.d/0-bootfiles
 }
 
+function pre_update_jetson_fix {
+    log "Caching current extlinux.conf for ${SLUG} fix"
+    extlinux_path=/boot/extlinux
```

Comment:
we should ensure that `/boot/extlinux` is the exactly correct location. I know there was some movement of this file from boot to rootfs or something, so worth checking with @acostach the exact l location this file needs to end up in the 2.45.1 OS version to ensure it works